src/deepak.py

```diff
@@ -0,0 +1,2 @@
+def add(a,b):
```

Comment:
Please add docstring at the First line of the file. And also add doc string to the function.